_posts/03-05-01-Command-Line-Interface.md

```diff
@@ -53,9 +53,9 @@ Hello, world
 
  * [Learn about running PHP from the command line][php-cli]
 
-[phpinfo]: https://secure.php.net/function.phpinfo
-[cli-options]: https://secure.php.net/features.commandline.options
-[argc]: https://secure.php.net/reserved.variables.argc
-[argv]: https://secure.php.net/reserved.variables.argv
+[phpinfo]: https://www.php.net/function.phpinfo
+[cli-options]: https://www.php.net/features.commandline.options
+[argc]: https://www.php.net/reserved.variables.argc
+[argv]: https://www.php.net/reserved.variables.argv
 [exit-codes]: https://www.gsp.com/cgi-bin/man.cgi?section=3&amp;topic=sysexits
-[php-cli]: https://secure.php.net/features.commandline.options
+[php-cli]: https://www.php.net/manual/en/features.commandline.php
```

Comment:
Good catch to link to the chapter instead of the subsection. :+1:

Reply:
Uh oh... but it's now no longer a language agnostic link... I think this needs to be fixed to https://www.php.net/features.commandline

Reply:
@jrfnl maybe. PHP site doesn't redirect and differentiate between these two. When clicking through the PHP manual or on Google searches the URL with `.php` is used there. So whichever is used is fine, I guess.

Reply:
Well, my reason for flagging it was two-fold: the contributing guide states to use language agnostic links + will make life easier on translators to use language agnostic links.

Reply:
You mean URL without the `en`?

Reply:
Ah, I've missed that you've written the URL above. Then I suggest to send PR for this.